app/scripts/board/board.controller.js

```diff
@@ -7,9 +7,76 @@
 
 	BoardController.$inject = [
 		'$scope',
-		'board'];
+		'toaster',
+		'ListWizard',
+		'List',
+		'board',
+		'lists'];
```

Comment:
They are sorted - it's just not alphabetically. First goes core angular depencdencies (scope always first) like $timeout. Then external angular depencdencies like $state, then external modules like $modal. Next are internal services and last but not least - resolved variables